docs/user/kafka/configuration.rst

```diff
@@ -0,0 +1,20 @@
+Configuration
+=============
+
+Use of the Kafka data store does not require server-side configuration,
```

Comment:
What Kafka configuration is in the geomesa-site.xml?  Is it worth mentioning those options to motivate checking it out for the KDS user?